### ISSUE TITLE
Missing YAML mapping documentation

### ISSUE THREAD

**Issue by belinde** [doctrine/doctrine-website#45]
The YAML mapping documentation got lost in the new (very pretty) website. This page lead to a 404 Not Found: https://www.doctrine-project.org/projects/doctrine-orm/en/stable/reference/yaml-mapping.html

**Comment by Ocramius** [comment]
Yaml mappings are gone from `master`, so is the YAML driver itself.

**Comment by Majkl578** [comment]
_Stable_ points to 2.6 and it's missing there, it's not working in 2.5 either.

**Comment by belinde** [comment]
That's frankly alarming: I've used yaml in every project since EVER, and I really dislike annotations 
Will it be possible to continue using yaml, in some way?

**Comment by Ocramius** [comment]
See https://github.com/doctrine/DoctrineBundle/issues/776 about that (concise explanation).

My tip is to simply migrate to XML mappings.

**Comment by belinde** [comment]
Thanks a lot, I've already migrated one project to XML and it was quite painless, and now I wonder how I lived before without autocompletion :-/ 

Now I see that YAML documentation has come back to life: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/yaml-mapping.html
Can I suggest to add a deprecation notice on it? I think that there are a lot o developers like me, in the wild, that will panick without YAML. Also integrating the issue 776 linked by @Ocramius coul be very useful to calm the spirits.

As an unrelated note, thank you all for your precious work!
